test/mocha/cli.js

```diff
@@ -49,4 +49,25 @@ describe("bin/uglifyjs", function () {
             done();
         });
     });
+    it("Should append source map to output when using --source-map-inline", function (done) {
+       var command = uglifyjscmd + ' test/input/issue-1323/sample.js --source-map-inline';
+
+       exec(command, function (err, stdout) {
+           if (err) throw err;
+
+           assert.strictEqual(stdout, "var bar=function(){function foo(bar){return bar}return foo}();\n" + 
```

Comment:
Nope. Old school.

You could make another PR for it, but not sure it would be accepted. Not all semicolons are used consistently, etc. Formatting PRs that add no value are routinely rejected.